data-hibernate-jpa/build.gradle

```diff
@@ -9,7 +9,6 @@ dependencies {
 
     api projects.dataRuntime
     api projects.dataModel
-    api libs.hibernate.core
```

Comment:
how would one reference the hibernate API?

Reply:
It's referenced from the SQL Hibernate module

Reply:
Hmm but it might need to be on `api`?

Reply:
right, need to be careful as it might cause compile errors in user projects

Reply:
Fixed